includes/functions.php

```diff
@@ -1342,7 +1360,8 @@ function tsml_import_buffer_set($meetings, $data_source_url = null, $data_source
 {
 	global $tsml_programs, $tsml_program, $tsml_days, $tsml_meeting_attendance_options, $tsml_data_sources;
 
-	if (strpos($data_source_url, "sheets.googleapis.com") !== false) {
+	//if (strpos($data_source_url, "sheets.googleapis.com") !== false) { refactored to be PHP 8.0 compliant
+	if (str_contains($data_source_url, "sheets.googleapis.com") !== false) {
```

Comment:
I'm still getting deprecated notices on this line. Changing this line to the following worked to remove the notice:
`if (!empty($data_source_url) && str_contains($data_source_url, "sheets.googleapis.com") !== false) {`

Reply:
This is outside the scope of this PR, but I agree with implementing this fix to get rid of those annoying deprecation notices.